src/Models/HydrostaticFreeSurfaceModels/compute_hydrostatic_free_surface_tendencies.jl

```diff
@@ -154,7 +154,7 @@ function compute_free_surface_tendency!(grid, model, kernel_parameters)
 
     launch!(arch, grid, kernel_parameters,
             compute_hydrostatic_free_surface_Gη!, model.timestepper.Gⁿ.η, 
-            grid, args)
+            grid, args...)
```

Comment:
Does this produce a catastrophic slow down or no?

Reply:
so this is now being passed as var-args, which may resolve but yeah needs testing

Reply:
Ok, I see the rationale so I think just a test or three (for this we can simply run the examples and show that `run!(simulation)` takes the same amount of time, or use an example setup but benchmark `time_step(model, dt)`).